src/components/ConversationSettings/ConversationAvatarEditor.vue

```diff
@@ -0,0 +1,310 @@
+<!--
+	- @copyright 2022 Christopher Ng <chrng8@gmail.com>
+	-
+	- @author Christopher Ng <chrng8@gmail.com>
+	- @author Marco Ambrosini <marcoambrosini@icloud.com>
+
+	- @license AGPL-3.0-or-later
+	-
+	- This program is free software: you can redistribute it and/or modify
+	- it under the terms of the GNU Affero General Public License as
+	- published by the Free Software Foundation, either version 3 of the
+	- License, or (at your option) any later version.
+	-
+	- This program is distributed in the hope that it will be useful,
+	- but WITHOUT ANY WARRANTY; without even the implied warranty of
+	- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	- GNU Affero General Public License for more details.
+	-
+	- You should have received a copy of the GNU Affero General Public License
+	- along with this program. If not, see <http://www.gnu.org/licenses/>.
+	-
+-->
+
+<template>
+	<section id="vue-avatar-section">
+		<div v-if="!showCropper" class="avatar__container">
+			<div class="avatar__preview">
+				<ConversationIcon v-if="!loading"
+					:item="conversation"
+					:is-big="true"
+					:disable-menu="true" />
+				<div v-else class="icon-loading" />
+			</div>
+			<div class="avatar__buttons">
+				<NcButton :aria-label="t('settings', 'Upload profile picture')"
+					@click="activateLocalFilePicker">
+					<template #icon>
+						<Upload :size="20" />
+					</template>
+				</NcButton>
+				<NcButton :aria-label="t('settings', 'Choose profile picture from files')"
+					@click="openFilePicker">
+					<template #icon>
+						<Folder :size="20" />
+					</template>
+				</NcButton>
+				<NcButton v-if="hasAvatar"
+					:aria-label="t('settings', 'Remove profile picture')"
+					@click="removeAvatar">
+					<template #icon>
+						<Delete :size="20" />
+					</template>
+				</NcButton>
+			</div>
+			<span>{{ fileTypeWarning }}</span>
+			<input :id="inputId"
```

Comment:
Should have a class for ...

Reply:
What do you mean here @ShGKme ?

Reply:
Second part of a question is in the next comment 😀

I meant, shouldn't we add a class for styling this span?